src/applications/facility-locator/components/search-results-items/Covid19Result.jsx

```diff
@@ -9,18 +9,26 @@ import LocationAddress from './common/LocationAddress';
 import LocationOperationStatus from './common/LocationOperationStatus';
 import LocationDistance from './common/LocationDistance';
 import CovidPhoneLink from './common/Covid19PhoneLink';
+import recordEvent from 'platform/monitoring/record-event';
 
-const Covid19Result = ({ location, index }) => {
+const Covid19Result = ({
+  location,
+  index,
+  showCovidVaccineSchedulingLinks,
```

Comment:
you could probably update `showCovidVaccineSchedulingLinks ` in the `propTypes`

Reply:
Good catch, thanks!